setup-macos.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Base URL for downloads
+BASE_URL="https://storage.googleapis.com/chrome-for-testing-public/123.0.6312.105"
```

Comment:
I was not able to get the chrome binary working either. It ran locally when I commented out the chrome binary reference in defaults.

I had it commented out in my first PR but changed it back because I still must still be used or it'll be a breaking change with other environments. One option would be to add a conditional for mac not to use the chrome binary in defaults. 

I will do some more troubleshooting this afternoon, but that was a known error I shared with Lyie.